dracut-posttrans

```diff
@@ -19,6 +19,7 @@ initrd_regenerate() {
 
 initrd_regenerate_all() {
 	"$DRACUT" -f --regenerate-all
+	return $?
```

Comment:
This is superfluous, the function will return the exit status of dracut anyway.

Reply:
Sure, I just wanted to follow the same style as the `initrd_regenerate()` function above:

https://github.com/openSUSE/suse-module-tools/blob/71e6c1a96cf75eeb54a5b9468ea02421f8b23e01/dracut-posttrans#L17

Removing both.